### DISCUSSION TITLE
Is there a compatibility issue with Security (S0 / S2) and Aeotec devices?

### DISCUSSION THREAD

**Discussion by austwhite** [zwave-js/zwave-js#5926] [General]
I am struggling with AEOTEC devices (Switch 6 and Nano Dimmer) not connecting securely, to Z-Wave JS, despite me following the correct process on the device (double press of pairing button, LED flashing blue for pairing securely).
ZWave JS reports security none (Timeout) every time.
Is this something that may be a known issue?

Using latest ZWave-JS-UI on a Raspberry Pi 3B+ with a Zwave.Me Razberry 7 Hat.  
Nanodimmer is on Firmware 2.4 (latest)
Switch 6 is on Firmware 1.4 (latest)
Both devices I believe are ZWave 500 series devices from what I can get from the specs.   The Razberry 7 is a 700 series controller.

**Comment by AlCalzone** [comment]
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

That log should capture a failed inclusion attempt.

That said, there are no known issues regarding this. Usually this is an indication for bad connectivity.

**Comment by austwhite** [comment]
I'll get onto a log tomorrow for you.   I have a Fibaro and a Remotec device that both connect S0 no issues (these two don't do S2) but none of the aeotecs are doing it. - All directly connect to the controller as they are not far away :).
Anyway, I'll get that log you want tomorrow as it is 9pm Saturday here in Australia (Mlebourne)   :)

**Comment by austwhite** [comment]
Well, I might as well close this as I re-added the devices today and it all worked.    I have no idea what was causing the issue, but all the Aeotec devices just worked as expected, every one successfully connected using proper security.

Thanks for the help.  Maybe it was just some kind of transiant interference.
